finality-aleph/src/nodes/validator_node.rs

```diff
@@ -42,10 +48,40 @@ where
         millisecs_per_block,
         justification_rx,
         backup_saving_path,
+        external_addresses,
+        validator_port,
         ..
     } = aleph_config;
 
-    let (validator_network, validator_network_identity) = tcp_network::new_noop().await;
+    // We generate the phrase manually to only save the key in RAM.
```

Comment:
Mostly to avoid spamming relatively unimportant files on disk right next to the absolutely crucial aleph validator keys. Expanded the comment to make this clearer.